AutoRest/Generators/NodeJS/NodeJS/Templates/ModelTemplateTS.cshtml

```diff
@@ -22,7 +22,7 @@ export interface @(Model.Name) {
     var propertyList = new List<Property>(Model.ComposedProperties);
     for (int i = 0; i < propertyList.Count; i++)
     {
-    @:@(Model.PropertyTS(propertyList[i], true));
+    @:@(ModelTemplateModel.PropertyTS(propertyList[i], true));
```

Comment:
I didn't really want to change it -- this happened when I did a pull from "dev" branch and ran "gulp".  Gulp complained that PropertyTS could be a static method, so I had to make it a static method (and thus change this) in order to get gulp to proceed.